tests/rest-api/methods.yml

```diff
@@ -203,6 +203,227 @@
         }
     ]
     }
+- uri: /v2/apps?embed=apps.taskStats
+  method: GET
+  content: |
+    {
+    "apps": [
+        {
+            "args": null,
```

Comment:
tried to reduce further than this, but general app deserialization fails at that point.